Jenkinsfile_CNP

```diff
@@ -63,6 +63,5 @@ withPipeline("java", product, component) {
 
   loadVaultSecrets(secrets)
   enableSlackNotifications(notificationsChannel)
-  enableDockerBuild()
```

Comment:
what did it do? do we need some new configuration?

Reply:
The message from Jenkins is:
```
enableDockerBuild() is deprecated, a Dockerfile has been mandatory since 17/12/2019, please remove this option from your Jenkinsfile This configuration will stop working by 18/02/2020 00:00 AM ( in 11 days )
```
So it looks like it was doing something that is now the default anyway.